example/ck_tile/01_fmha/fmha_fwd.hpp

```diff
@@ -510,8 +510,12 @@ auto fmha_fwd_splitkv_create_kargs_and_grids(fmha_fwd_splitkv_args args)
         }
     }();
 
-    dim3 grids =
-        Kernel::GridSize(args.batch, args.nhead_q, args.max_seqlen_q, args.hdim_v, args.num_splits);
+    const ck_tile::index_t nhead =
+        (Kernel::kMergeNumHeadGroupsSeqLenQ ? args.nhead_k : args.nhead_q);
+    const ck_tile::index_t max_seqlen_q =
+        args.max_seqlen_q * (Kernel::kMergeNumHeadGroupsSeqLenQ ? args.nhead_q / args.nhead_k : 1);
+
+    dim3 grids = Kernel::GridSize(args.batch, nhead, max_seqlen_q, args.hdim_v, args.num_splits);
 
```

Comment:
Suggest to pass both `nhead_q`,  `nhead_kv` to the GridSize,  rather than let upper layer to make a  less-specific  `nhead`.